repo/content/committed_content_index.go

```diff
@@ -10,6 +11,11 @@ import (
 	"github.com/kopia/kopia/repo/blob"
 )
 
+// combineSmallIndexThreshold is the count threshold below which indexes
```

Comment:
What about `smallIndexEntryCountThreshold`?

```suggestion
// smallIndexEntryCountThreshold is the threshold to determine whether an
// index is small. Any index with fewer entries than this threshold 
```